README.md

```diff
@@ -31,32 +31,29 @@ installed).
 
 The user discussion and development of Git take place on the Git
 mailing list -- everyone is welcome to post bug reports, feature
-requests, comments and patches to git@vger.kernel.org (read
+requests, comments, and patches to git@vger.kernel.org (read
 [Documentation/SubmittingPatches][] for instructions on patch submission).
 To subscribe to the list, send an email with just "subscribe git" in
 the body to majordomo@vger.kernel.org. The mailing list archives are
 available at <https://lore.kernel.org/git/>,
-<http://marc.info/?l=git> and other archival sites.
+<http://marc.info/?l=git>, and other archival sites.
 
-Issues which are security relevant should be disclosed privately to
+Issues that are security relevant should be disclosed privately to
 the Git Security mailing list <git-security@googlegroups.com>.
 
 The maintainer frequently sends the "What's cooking" reports that
 list the current status of various development topics to the mailing
-list.  The discussion following them give a good reference for
-project status, development direction and remaining tasks.
+list.  The discussion following them gives a good reference for
+project status, development direction, and remaining tasks.
 
 The name "git" was given by Linus Torvalds when he wrote the very
 first version. He described the tool as "the stupid content tracker"
 and the name as (depending on your mood):
 
- - random three-letter combination that is pronounceable, and not
-   actually used by any common UNIX command.  The fact that it is a
+ - random three-letter combination that is pronounceable, and not actually used by any common UNIX command.  The fact that it is a
    mispronunciation of "get" may or may not be relevant.
- - stupid. contemptible and despicable. simple. Take your pick from the
-   dictionary of slang.
- - "global information tracker": you're in a good mood, and it actually
-   works for you. Angels sing, and a light suddenly fills the room.
+ - stupid. contemptible and despicable. simple. Take your pick from the dictionary of slang.
```

Comment:
Again, this only turns a correctly-wrapped bullet point into a single line that is too long.

Mind you, there is quite the room for improvement in this particular text. It is quite offensive and off-putting to casual readers. It would be a worthy goal to improve that, but that is not what this patch does.